Dockerfile

```diff
@@ -3,6 +3,11 @@ FROM praekeltfoundation/django-bootstrap:py3.7-stretch
 ENV DJANGO_SETTINGS_MODULE "healthcheck.settings.production"
 CMD ["healthcheck.wsgi:application"]
 
+RUN apt-get update && apt-get install -y gcc  \
+    && pip install zbar-py==1.0.4 \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get purge -y --auto-remove gcc
```

Comment:
There are `apt-get-install.sh` and `apt-get-purge.sh` scripts in the base image for this purpose.

The other thing we can do is to add it to https://github.com/praekeltfoundation/debian-wheel-mirror , so that there's a wheel already built for us, then we don't need this step.